### PR TITLE
Upgrade service fabric tools version (VS2017 latest seems to be fixed)

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

File: src/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication.sfproj [hunk 1]
```diff
@@ -3,9 +3,8 @@
   <Import Project="..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.5.0\build\Microsoft.VisualStudio.Azure.Fabric.Application.props" Condition="Exists('..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.5.0\build\Microsoft.VisualStudio.Azure.Fabric.Application.props')" />
   <PropertyGroup Label="Globals">
     <ProjectGuid>6722aac7-f141-4492-85bb-be0c82ef7f98</ProjectGuid>
-    <ProjectVersion Condition="$(VisualStudioVersion) == '14.0'">1.5</ProjectVersion>
-    <MinToolsVersion Condition="$(VisualStudioVersion) == '14.0'">1.5</MinToolsVersion>
-    <ProjectVersion Condition="$(VisualStudioVersion) == '15.0'">1.4</ProjectVersion>
+    <ProjectVersion>1.5</ProjectVersion>
```
Comment (SeanFeldman) [inline src/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication.sfproj]: Are you sure we should remove vs15 safety?

Comment (danielmarbach) [inline src/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication.sfproj]: Works on my machine now with latest VS 🗡

Comment (danielmarbach) [inline src/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication.sfproj]: I'm going to ping the channel to do VS upgrade before working in the persister.

Comment (SeanFeldman) [inline src/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication/NServiceBus.Persistence.ServiceFabric.AcceptanceTestsApplication.sfproj]: 👍

**Pull request merged**